jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/DeleteResourceDialog.kt

```diff
@@ -33,6 +34,14 @@ class DeleteResourceDialog(
             row {
                 deleteResourceConfirmation(grow)
             }
+            createNoteOrCommentRow(
+                ComponentPanelBuilder.createCommentComponent(
+                    message("sqs.delete.warning_text"),
```

Comment:
This is a generic delete component right? We should probably not put SQS specific logic in here, we should find a way to extend it so that other resource types can (optionally) provide information if required.

Reply:
Changed it to a more generic format which could be used by multiple resource types and the information in the comment can be custom to the resourceType if required